.github/workflows/publish-stage.yaml

```diff
@@ -3,7 +3,7 @@ name: publish-stage
 on:
   push:
     branches:
-      - "main"
```

Comment:
This branch is going to be deployed to `stage.unsupported.docs.camunda.io/8.1`, and we only want to deploy this branch.